simplify url_dictionary & add urls for global site

Lukom opened the pull request:
simplify url_dictionary & add urls for global site

Lukom committed:
simplify url_dictionary & add urls for global site

SqREL commented on url_dictionary.gemspec:
```diff
@@ -1,20 +1,19 @@
-  spec.authors       = ["Niels Buus"]
-  spec.email         = ["nb@lokalebasen.dk"]
+  spec.authors       = ["Lev Lukomskyi"]
+  spec.email         = ["ll@matchoffice.com"]
```
what the hell?

SqREL replied:
it's an array, you can add yourself

Lukom replied:
hehe, because I've rewritten all I thought maybe I can change author.. But yeah, maybe not. Maybe it would be better to just change it to "MatchOffice" to not bother with this data.

arfelio committed:
add two locations kinds

arfelio committed:
add new kinds for all languages

arfelio committed:
Merge pull request #34 from arfelio/ao/add_location_kinds

add two locations kinds

starling14 committed:
Add sections

Add sections only for dk because no plans for sale on other sites now.
We need this chapter for friendly urls path path.

starling14 committed:
Merge branch 'staging' of github.com:lokalebasen/url_dictionary into staging

arfelio committed:
Merge pull request #35 from starling14/staging

Staging

starling14 committed:
Refactor section 'user' to 'own_use'

starling14 committed:
Merge pull request #36 from starling14/staging

Refactor section 'user' to 'own_use'

Lukom committed:
put version in version.rb for bumper

Lukom committed:
rename uk => gb; leave uk key temporarily just in case

Lukom committed:
Merge remote-tracking branch 'upstream/master' into site_key_feature
